### PR TITLE
#5231 Media type fixes

### LINKED ISSUE #5231: Music cards fix
image

- I think we should first fix it and then we can redesign it
- #5228 

same with nft detail 
image
- it should be square

weird that on explore it has a thumbnail
image

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #5231
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screen Shot 2023-03-12 at 3 48 55 PM
Screen Shot 2023-03-12 at 3 49 00 PM

### CODE REVIEW

**Review by roiLeo — COMMENTED** [review]
I would put Waveform at top & maybe remove player (or just show time)

File: components/explore/SidebarFilter.vue [hunk 1]
```diff
@@ -30,5 +30,6 @@ const open = computed(
   position: sticky;
   top: 72px;
   height: calc(100vh - 72px);
+  margin-right: 32px;
```
Comment (roiLeo) [inline components/explore/SidebarFilter.vue:33]: ?? out of scope

File: layouts/explore-layout.vue [hunk 2]
```diff
@@ -77,4 +77,8 @@ hr {
     padding: 0 1rem;
   }
 }
+
+.explore-header {
+  padding-bottom: 40px;
+}
```
Comment (roiLeo) [inline layouts/explore-layout.vue:83]: ?? out of scope

**Review by roiLeo — CHANGES_REQUESTED** [review]

File: components/explore/SidebarFilter.vue (hunk 1 above)
Comment (vikiival) [reply components/explore/SidebarFilter.vue:33]: ups looks like two PRs we married together (sending flowers and wedding cake)
It is from different PR from @prachi00

Comment (prachi00) [reply components/explore/SidebarFilter.vue:31]: my bad, I am closing this one, I raised another clean pr #5262

**Pull request closed without merging**